sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/IndexingPolicy.java

```diff
@@ -4,22 +4,28 @@
 package com.azure.cosmos.models;
 
 import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
```

Comment:
I think you should also be able to revert all changes to this file

Reply:
Done.